content/docs/glossary/sponsored-reserves.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Sponsored Reserves
+order:
+---
+
+Protocol 14 introduces operations that allow an account to pay the base reserves for another account. This is done by using the [Begin Sponsoring Future Reserves](../start/list-of-operations.mdx#begin-sponsoring-future-reserves) and [End Sponsoring Future Reserves](../start/list-of-operations.mdx#end-sponsoring-future-reserves) operations.
+
+The sponsoring account establishes the is-sponsoring-future-reserves-for relationship, and the sponsored account terminates it. While this relationship exists, reserve requirements that would normally accumulate on the sponsored account will now accumulate on the sponsoring account. Both operations must appear in a single transaction, which guarantees that both the sponsoring and sponsored accounts agree to every sponsorship.
+
+## Sponsorship effect on Minimum Balance
+
+The [Minimum Balance](./minimum-balance.mdx) calculation once sponsorships are introduced becomes `(2 + numSubEntries + numSponsoring - numSponsored) * baseReserve + liabilities.selling`.
+
+When account `A` is-sponsoring-future-reserves-for account `B`, any reserve requirements that would normally accumulate on `B` will instead accumulate on `A` as reflected in `numSponsoring`. The fact that these reserves are being provided by another account will be reflected on `B` in `numSponsored`, which will cancel out the increase in `numSubEntries`, keeping the minimum balance unchanged for `B`.
+
+When a sponsored ledger entry or sub-entry is removed, `numSponsoring` is decreased on the sponsoring account and `numSponsored` is decreased on the sponsored account.
+
+## What can be sponsored?
+
+Anything that increases the minimum balance can be sponsored (Accounts, Offers, Trustlines, AccountData, and Signers).
+
+## Claimable Balances
+
+[Claimable Balances](./claimable-balance.mdx) are unique in that they must be sponsored. They are not sub-entries of an account, so the sponsoring account uses the sponsorship mechanism to pay the base reserve by increasing `numSponsoring`. The sponsorship logic is handled through the Claimable Balance operations, so the use of sponsorships is transparent to the user.
+
+## Relevant operations
+
+### Begin and end sponsorships
+
+[Begin Sponsoring Future Reserves](../start/list-of-operations.mdx#begin-sponsoring-future-reserves) will establish the is-sponsoring-future-reserves-for relationship where the sponsoring account is the source account of the operation, and the account specified in the operation is the sponsored account.
+
+[End Sponsoring Future Reserves](../start/list-of-operations.mdx#end-sponsoring-future-reserves) will end the current is-sponsoring-future-reserves-for relationship for the source account of the operation.
+
+At the end of any transaction, there must be no ongoing is-sponsoring-future-reserves-for relationships. This is why these two operations must be used together in a single transaction.
```

Comment:
Not sure if the preview is out of date, but it looks like headers aren't rendering exactly right [here](https://developers-pr244.previews.kube001.services.stellar-ops.com/docs/glossary/sponsored-reserves/#relevant-operations) and elsewhere on the page.

Reply:
The preview doesn't seem to be updating for some reason. @rice2000 any idea why?